assets/blocks/quiz/index.js

```diff
@@ -0,0 +1,45 @@
+import { InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import registerSenseiBlocks from '../register-sensei-blocks';
+import { EditQuizBlock as edit } from './edit';
+import questionBlock from './question-block';
+import metadata from './block.json';
+
+/**
+ * Quiz block definition.
+ */
+const quizBlock = {
+	...metadata,
+	title: __( 'Quiz', 'sensei-lms' ),
+	description: __(
+		'A collection of questions students need to answer.',
+		'sensei-lms'
+	),
+	keywords: [
+		__( 'Quiz', 'sensei-lms' ),
```

Comment:
We could also add 'Exam' here. Also I don't think that Quiz is needed as it's the same with the name.

Reply:
👍 09b13f8